web/styles/zulip.css

```diff
@@ -666,6 +666,14 @@ li,
     }
 }
 
+#copy_generated_link_container {
+    display: flex;
+    width: 100%;
+    margin-right: -32px;
```

Comment:
I suspect you can add another commit to get rid of the -32px margin both here and on `copy_generated_invite_link` with a bit more effort; can you take care of that while you're fiddling with this?

It might involve not swapping the HTML order, which would be more natural.

Reply:
Sure

Reply:
I have looked into it. That margin right is required due to the padding from the parent container - `dialog error `. So need to modify it and I am not sure if it won't break styling anywhere else.

Reply:
If we change the HTML order,  `margin-top: -24px` must also be added which felt unnecessary.